IEXSharp/Service/Cloud/PremiumData/BrainCompany/BrainCompanyService.cs

```diff
@@ -0,0 +1,51 @@
+{
+	public class BrainCompanyService : IBrainCompanyService
+	{
+		private readonly ExecutorREST executor;
+
+		internal BrainCompanyService(ExecutorREST executor)
+		{
+			this.executor = executor;
+		}
+
+		public async Task<IEXResponse<IEnumerable<SentimentIndicatorResponse>>> ThirtyDaySentimentIndicatorAsync(string symbol) =>
+			await executor.NoParamExecute<IEnumerable<SentimentIndicatorResponse>>($"time-series/PREMIUM_BRAIN_SENTIMENT_30_DAYS/{symbol}");
+
+		public async Task<IEXResponse<IEnumerable<SentimentIndicatorResponse>>> SevenDaySentimentIndicatorAsync(string symbol) =>
+			await executor.NoParamExecute<IEnumerable<SentimentIndicatorResponse>>($"time-series/PREMIUM_BRAIN_SENTIMENT_7_DAYS/{symbol}");
+
+		public async Task<IEXResponse<IEnumerable<EstimatedReturnRankingResponse>>> TwentyOneDayMachineLearningEstimatedReturnRankingAsync(string symbol) =>
+			await executor.NoParamExecute<IEnumerable<EstimatedReturnRankingResponse>>($"time-series/PREMIUM_BRAIN_RANKING_21_DAYS/{symbol}");
+
+		public async Task<IEXResponse<IEnumerable<EstimatedReturnRankingResponse>>> TenDayMachineLearningEstimatedReturnRankingAsync(string symbol) =>
+			await executor.NoParamExecute<IEnumerable<EstimatedReturnRankingResponse>>($"time-series/PREMIUM_BRAIN_RANKING_10_DAYS/{symbol}");
+
+		public async Task<IEXResponse<IEnumerable<EstimatedReturnRankingResponse>>> FiveDayMachineLearningEstimatedReturnRankingAsync(string symbol) =>
+			await executor.NoParamExecute<IEnumerable<EstimatedReturnRankingResponse>>($"time-series/PREMIUM_BRAIN_RANKING_5_DAYS/{symbol}");
+
+		public async Task<IEXResponse<IEnumerable<EstimatedReturnRankingResponse>>> ThreeDayMachineLearningEstimatedReturnRankingAsync(string symbol) =>
+			await executor.NoParamExecute<IEnumerable<EstimatedReturnRankingResponse>>($"time-series/PREMIUM_BRAIN_RANKING_3_DAYS/{symbol}");
+
+		public async Task<IEXResponse<IEnumerable<EstimatedReturnRankingResponse>>> TwoDayMachineLearningEstimatedReturnRankingAsync(string symbol) =>
+			await executor.NoParamExecute<IEnumerable<EstimatedReturnRankingResponse>>($"time-series/PREMIUM_BRAIN_RANKING_2_DAYS/{symbol}");
+
+		public async Task<IEXResponse<IEnumerable<LanguageMetricsOnCompanyFilingsResponse>>> LanguageMetricsOnCompanyFilingsQuarterlyAndAnnualAsync(string symbol) =>
+			await executor.NoParamExecute<IEnumerable<LanguageMetricsOnCompanyFilingsResponse>>($"time-series/PREMIUM_BRAIN_LANGUAGE_METRICS_ALL/{symbol}");
+
+		public async Task<IEXResponse<IEnumerable<LanguageMetricsOnCompanyFilingsResponse>>> LanguageMetricsOnCompanyFilingsAnnualOnlyAsync(string symbol) =>
+			await executor.NoParamExecute<IEnumerable<LanguageMetricsOnCompanyFilingsResponse>>($"time-series/PREMIUM_BRAIN_LANGUAGE_METRICS_10K/{symbol}");
+
+		public async Task<IEXResponse<IEnumerable<DifferencesInLanguageMetricsOnCompanyFilingsResponse>>> DifferencesInLanguageMetricsOnCompanyFilingsQuarterlyAndAnnualFromPriorPeriodAsync(string symbol) =>
```

Comment:
Yes, these are getting far too long, however they are named consistently with all of the other premium, (and I assume IEX points). ie. Their names are the titles of the endpoint document (Usually less the premium provider name) - https://iexcloud.io/docs/api/#brain-companys-language-metrics-on-company-filings-annual-only

Reply:
Nothing we can do about their naming conventions. Shortening them would just add to opportunity for confusion.

Reply:
I agree, consistency is key.